ginisdk/src/androidTest/java/net/gini/android/SdkIntegrationTest.java

```diff
@@ -274,7 +280,214 @@ public void testPublicKeyPinningWithMultiplePublicKeys() throws Exception {
+                })
+                .onSuccessTask(new Continuation<Document, Task<Document>>() {
+                    @Override
+                    public Task<Document> then(final Task<Document> task) throws Exception {
+                        partialDocuments.add(task.getResult());
+                        return documentTaskManager.createPartialDocument(page3, "image/png", null, null);
+                    }
+                })
+                .onSuccessTask(new Continuation<Document, Task<Document>>() {
+                    @Override
+                    public Task<Document> then(final Task<Document> task) throws Exception {
+                        partialDocuments.add(task.getResult());
+                        final LinkedHashMap<Document, Integer> documentRotationDeltaMap = new LinkedHashMap<>();
+                        for (final Document partialDocument : partialDocuments) {
+                            documentRotationDeltaMap.put(partialDocument, 0);
+                        }
+                        return documentTaskManager.createCompositeDocument(documentRotationDeltaMap, null);
+                    }
+                }).onSuccessTask(new Continuation<Document, Task<Document>>() {
+                    @Override
+                    public Task<Document> then(final Task<Document> task) throws Exception {
+                        compositeDocument.set(task.getResult());
+                        return documentTaskManager.pollDocument(task.getResult());
+                    }
+                }).onSuccessTask(new Continuation<Document, Task<Map<String, SpecificExtraction>>>() {
+                    @Override
+                    public Task<Map<String, SpecificExtraction>> then(final Task<Document> task) throws Exception {
+                        return documentTaskManager.getExtractions(task.getResult());
+                    }
+                });
+        task.waitForCompletion();
+
+        assertEquals(3, partialDocuments.size());
+        final Map<String, SpecificExtraction> extractions = task.getResult();
+        assertNotNull(extractions);
+
+        assertEquals("IBAN should be found", "DE96490501010082009697", extractions.get("iban").getValue());
+        assertEquals("Amount to pay should be found", "145.00:EUR", extractions.get("amountToPay").getValue());
+        assertEquals("BIC should be found", "WELADED1MIN", extractions.get("bic").getValue());
+        assertEquals("Payee should be found", "Mindener Stadtwerke GmbH", extractions.get("paymentRecipient").getValue());
+        assertEquals("Payment reference should be found", "ReNr TST-00019, KdNr 765432", extractions.get("paymentReference").getValue());
+
+        // all extractions are correct, that means we have nothing to correct and will only send positive feedback
+        // we should only send feedback for extractions we have seen and accepted
+        Map<String, SpecificExtraction> feedback = new HashMap<String, SpecificExtraction>();
```

Comment:
Why don't you test this in a different test?

Reply:
🤦‍♂️